server/bundler/loader.js

```diff
@@ -92,6 +99,16 @@ function requireTemplate( module ) {
 	return 'require( ' + JSON.stringify( module ) + ' )();\n';
 }
 
+function singleEnsure( chunkName ) {
+	var result = [
+		'case ' + JSON.stringify( chunkName ) + ':',
+		'	return require.ensure([], function() {}, ' + JSON.stringify( chunkName ) + ' );',
```

Comment:
I wonder whether we should add to `_loadedSections` here, so the loading `setState` and `require.ensure` don't run needlessly. We'd probably have to extract the error handling, though.